April23/update-cycle.md

```diff
@@ -0,0 +1,62 @@
+---
+title: The Effect of Mojang's New Update Cycle on Modders
+author: Zxhir
+---
+
+# Introduction
+
+Modding has been an integral part of the Minecraft community since its inception. One of the reasons for its success is the ability for players to create their own modifications, or "mods," that change the game in various ways. These mods can range from simple cosmetic changes to complete overhauls of the game's mechanics. They are often created by independent developers who work tirelessly to improve the game for others. However, the recent update cycle by Mojang has had a significant impact on modders, and many are struggling to keep up with the changes.
+
+The new update cycle, which pushes out small updates more frequently, has made it difficult for modders to maintain their mods. When Mojang changes the game, it can interfere with the modders' code. They must tweak their code to work with the new update. This process can be time-consuming and frustrating, and it can also make mods useless and broken if Minecraft cannot recognize the input.
+
+While the new update cycle has been beneficial for vanilla players and the development team, it has had a negative impact on modders. In this article, we will explore the effect of the update cycle on modders and how it has impacted the Minecraft modding community. We will also look at the challenges modders face and the potential solutions that can help them overcome them.
+
+## Mojang's Update Cycle
+
+The new update cycle pushes out small updates more frequently for versions like 1.19.3. This is helpful for vanilla players and probably for the dev team, but it has a significant impact on modders. The modding community or players often rely on mods that are outdated and no longer work with the latest version of the game. This means that modders have to constantly update their mods to be compatible with the latest version of the game.
+
+Mojang releases updates for various reasons. These updates may include bug fixes, new features, and improvements to game performance. But modders often face challenges when Mojang changes their game and interferes with modder code. The modder must tweak their code until the game recognises the input. If Minecraft cannot recognize the input, it may crash the game or break, making the mod (and sometimes the game itself) useless and broken.
+
+## Modders and Minecraft Updates
+
+The modding decline in Minecraft is also attributed to the fact that there are fewer new players coming into the game, which means that there are fewer people interested in creating and using mods. Additionally, the introduction of the Minecraft Marketplace has made it difficult for modders to monetize their work, which has discouraged many from continuing to create mods for the game.
```

Comment:
Do we have a source for this?

Reply:
Personally I think it's a bit off topic. 

If we could just leave this conversation open for now and the next reviewer can give their opinion as well and we'll see